numberninecms/cms/0.1/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "NumberNine\\Bundle\\NumberNineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "public/": "%PUBLIC_DIR%/",
+        "docker/": "./docker/"
```

Comment:
Let's use "copy-from-package" for the public and docker folders. That will make it easier for you to keep them updated.